### DISCUSSION TITLE
Continue where I left off

### DISCUSSION THREAD

**Discussion by paivadaniel** [zidoro/pomatez#572] [Ideas]
Hello, imagine that I have completed 2 pomodoros of 50 minutes, and have completed 32 minutes of the third pomodoro, when suddenly, my computer crashes or the power goes out. When I turn my computer back on, I would like the application to start right where I left off, in session 3, with the time remaining. Is this possible to implement?

**Comment by sekwah41** [comment]
I'm planning to modify how some of the info is stored. Storing current sessions or start and pause times would make this possible. These changes are also needed for mobile too so will be coming eventually (hopefully soon if I have the time)
